packages/protocol/package.json

```diff
@@ -10,11 +10,17 @@
   "types": "src/index",
   "module": "src/index",
   "scripts": {
-    "clean": "rimraf lib tsconfig.tsbuildinfo",
+    "prepare": "yarn generate",
+    "clean": "rimraf lib tsconfig.tsbuildinfo && yarn generate clean",
     "build": "tsc",
     "build:clean": "yarn clean && yarn build",
     "type": "tsc --noEmit",
     "lint": "eslint --ext .ts,.tsx ./src",
-    "lint:fix": "eslint --fix --ext .ts,.tsx ./src"
+    "lint:fix": "eslint --fix --ext .ts,.tsx ./src",
+    "generate": "./schemaCodegen.sh"
```

Comment:
I'm still not fan of this script, we should use a node package / js script to download this schema. Windows users can not run bash scripts.

Reply:
I can try that, but at least our Windows users can easily run 'curl' or 'rm' . 
To mit it looks nicer to have a single central place to deal with the schema generation

Reply:
we still have, if we write this bash code in js, but then everybody can run this script.

Reply:
-> task for a future PR